ui/src/transaction.ts

```diff
@@ -395,6 +395,29 @@ const monthNames = [
   "December",
 ];
 
+export function getDay(date: Date): string {
+  return `${date.getUTCDate()}`;
+}
```

Comment:
This is to small of a function to have. I’d prefer to inline it. Plus, this should by `date.getDate` to display it in local time.

Reply:
Does that make sense since the getShortMonth is right below and both of these are always used in conjunction. It seems to me that either we move both or none.

Reply:
`getShortMonth` is used in two places so it makes sense to have this here. The whole setup is not ideal and we should start to use a library that deals with dates and time at some point. Until we get there we should try to avoid adding more abstraction and indirection. This is why I’d prefer to inline it.

Reply:
kk

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1900/commits/533e08baeb0e6e03c45f213979c8f96ab9bd9ccd